sunpy/map/sources/soho.py

```diff
@@ -279,6 +279,15 @@ def spatial_units(self):
 
         return SpatialPair(u.Unit(cunit1), u.Unit(cunit2))
 
+    @property
+    def unit(self):
+        bunit = self.meta.get('bunit', None)
+        if bunit is None:
+            return
+		# Maxwells aren't in the IAU unit style manual and therefore not a valid FITS unit
+		# The mapbase unit property forces this validation, so we must override it to prevent it.
```

Comment:
```suggestion
        # Maxwells aren't in the IAU unit style manual and therefore not a valid FITS unit
        # The mapbase unit property forces this validation, so we must override it to prevent it.
```